Libraries/WebSocket/__mocks__/event-target-shim.js

```diff
@@ -6,4 +6,13 @@
 // problem go away.
 
 'use strict';
-module.exports = function() {};
+
+function EventTarget() {
+  // Support both EventTarget and EventTarget([list, of, events])
+  // as a super class, just like the original module does.
+  if (arguments.length === 1 && Array.isArray(arguments[0])) {
```

Comment:
It mocks the `EventTarget` base class, which can be used either directly:
```js
class Foobar extends EventTarget { ... }
```
or you can call it as a function with an array of events you'd like your class to handle, in which case it'll return a new base class with automatically generated `onfoobar` getters/setters. Like so:
```js
class Foobar extends EventTarget(['message', 'error']) { ... }
```
This here just mocks that behaviour in a minimal way so the tests continue to work.